packages/react/src/components/Tab/Tab-test.js

```diff
@@ -21,38 +21,34 @@ describe('Tab', () => {
       expect(wrapper.hasClass('extra-class')).toBe(true);
     });
 
-    it('renders <a> with expected className', () => {
+    it('renders <buttton> with expected className', () => {
```

Comment:
probably want this to read `<button>` instead of `<buttton>` :)
```suggestion
    it('renders <button> with expected className', () => {
```